### ISSUE TITLE
On interval triggers execute regardless of connectors

### ISSUE THREAD

**Issue by katrinDY** [cortezaproject/corteza#694]
## Description
On interval triggers seem to execute regardless of being connected to WF steps. The wf here writes to the server logs and does so every minute, here are lines copied from the logs(note time):

"server_1     | 21:11:00.003     ERROR   workflow        ####################################"
"server_1     | 21:12:00.002     ERROR   workflow        ####################################"

It makes sense it executes every minute however should it also be executing the rest of the workflow when not connected.

![Image](https://user-images.githubusercontent.com/85161724/208376697-107ea79d-417a-4e5e-ae22-47617d9f1ac6.png)

## Notes
Ping Andre for any questions

**Comment by tjerman** [comment]
For now, this is **not** a bug and is in fact... a feature 🙃 

## TL;DR

* Workflows can be executed even if there is no trigger defined. True, it can only be executed via the API (or as a sub workflow) but it is still a case we wanted to support when designing the initial version.
* When a trigger is executed and is not connected to **any** step, the runner will default to the orphan step; **only** if a single orphan step is found (a step with no parents apart from workflow triggers).

## Concrete solution

Simply uncheck the enabled checkbox in the workflow trigger.
